environments/sre/servicemonitors/observatorium-jaeger.servicemonitor.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   - port: admin-http
   namespaceSelector:
     matchNames:
-    - telemeter-production
+    - '{{namespace}}'
```

Comment:
Hmm I’m not convinced about putting more layers of te playing into this repo. This is no longer usable by the community and also means we cannot use this manifests in e2e tests

Reply:
Good point. I'm going to create a dedicated directory for this specific use case and leave others as is.

Reply:
I've ended up making them parametric so anyone can easily override `environment` and `namespace` values downstream.

Reply:
I agree with @squat. The environment should be obsolete entirely, as the Prometheus scrape only their environment now.